src/main/scala/nl/knaw/dans/bag/v0/DansV0Bag.scala

```diff
@@ -210,8 +210,13 @@ class DansV0Bag private(private[v0] val locBag: LocBag) extends DansBag {
   /**
    * @inheritdoc
    */
-  override def fetchFiles: Seq[FetchItem] = {
-    locBag.getItemsToFetch.asScala.map(fetch => fetch: FetchItem)
+  override def fetchFiles: Seq[FetchItem] = locBag.getItemsToFetch.asScala.map(fetch => fetch: FetchItem)
+
+  /**
+   * @inheritdoc
+   */
+  def addFetchItem(url: URL, pathInData: Path): Try[DansV0Bag] = {
```

Comment:
would you mind adding `override` to all these new methods. That indicates that this method is defined in the trait above.